functions-and-operators/string-functions.md

```diff
@@ -115,11 +115,38 @@ Return the character for each integer passed.
 
 ### [`CHAR_LENGTH()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_char-length)
 
-Return number of characters in argument.
+The `CHAR_LENGTH()` function is used to get the total number of characters in a given argument as an integer.
+
+Examples:
+
+```sql
+SELECT CHAR_LENGTH("TiDB") AS LengthOfString;
+
++----------------+
+| LengthOfString |
++----------------+
+|              4 |
++----------------+
+```
+
+```sql
+SELECT CustomerName, CHAR_LENGTH(CustomerName) AS LenghtOfName FROM Customers;
+
++--------------------+--------------+
+| CustomerName       | LenghtOfName |
++--------------------+--------------+
+| Albert Einstein    |           15 |
+| Robert Oppenheimer |           18 |
++--------------------+--------------+
+```
+
+> **Note:**
+>
+> The preceding example operates under the assumption that there is a database with a table named `Customers` and a column inside the table named `CustomerName`.
```

Comment:
```suggestion
> The preceding examples operate under the assumption that there is a database with a table named `Customers` and a column inside the table named `CustomerName`.
```

Reply:
only this example
![image](https://github.com/pingcap/docs/assets/79440533/7f1b2ce0-63f7-4d81-87f6-c5c52c714b02)